Modules do not have release notes

rwmanos opened the pull request:
While installing the Web and Scripting Module (WSM), openQA fails with
'NO matching needles for release-notes-wsm'. However, this should be
skipped because there is not release note for WSM.

Also, I removed the bugzilla link because this is a comment in a
RESOLVED bug which does not add a lot of information. From what I
understood there is no release note for WE.

rwmanos commented:
Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/210
Working example: http://rwmanosvm2.suse.cz/tests/709

sysrich commented:
Is it acceptable that the WSM has no release notes? do ANY modules have a release notes? if no, then maybe it would be best to have a solution that stops openQA for looking for any release notes for modules, which still looking for release notes for Extensions/Addons

rwmanos commented:
Hi Richard, I installed SLES12SP1 with all modules (Web and Scripting, Legacy, Public Cloud, Toolchain, Advanced Systems Management, Certifications, Containers) and none has release notes.

Also, from those modules only the Web and Scripting and the Legacy have additional license agreement. This is already taken care in 'lib/registration.pm'

I changed the following:
`next if ($a eq 'we' || $a eq 'wsm')`
to
`next if grep { $a eq $_ } qw(we lgm asmm certm contm pcm tcm wsm);`

I also updated the needles
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/210

and now it's possible to install all modules!

This is the new working example: http://rwmanosvm2.suse.cz/tests/725

okurz commented on tests/installation/releasenotes.pm:
```diff
@@ -40,9 +40,10 @@ sub run() {
     if (@addons) {
         for my $a (@addons) {
-            next if ($a eq 'we');    # https://bugzilla.suse.com/show_bug.cgi?id=931003#c17
+            # no release-notes for WE and all modqules.
```
Typo in this line

rwmanos replied:
Hi Oliver, corrected. Thank you for spotting that!
